sdks/go/types/common/consts.go

```diff
@@ -3,3 +3,8 @@ package common
},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"Empty","Empty"]}},"height":32,"num_leaves":13,"_phantom":null},"fee_merkle_tree":{"root":"Empty","height":20,"num_leaves":0,"_phantom":null},"reward_merkle_tree_v1":{"root":"Empty","height":20,"num_leaves":0,"_phantom":null},"reward_merkle_tree_v2":{"root":"Empty","height":160,"num_leaves":0,"_phantom":null},"chain_config":{"chain_config":{"Left":{"chain_id":"999999999","max_block_size":"1000000","base_fee":"1","fee_contract":"0x8ce361602b935680e8dec218b820ff5056beb7af","fee_recipient":"0x0000000000000000000000000000000000000000","stake_table_contract":null}}}},"delta":{"fees_delta":["0xb0cfa4e5893107e2995974ef032957752bb526e9","0x0000000000000000000000000000000000000000"],"rewards_delta":[]},"vid_share":{"V0":{"view_number":13,"payload_commitment":"HASH~K-IZa5wZsJE7EdcI0lUM3w9bAQbErg7sKqB9KyQ8cmgO","share":{"index":4,"evals":"FIELD~AAAAAAAAAAD7","aggregate_proofs":"FIELD~AQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAQGY","evals_proof":{"pos":"FIELD~BAAAAAAAAAC3","proof":[{"Leaf":{"value":"FIELD~a28U5q8mJxhr-KVeR03PbvP2zoGvVg2ocexR62cBcanN","pos":"FIELD~BAAAAAAAAAC3","elem":"FIELD~AAAAAAAAAAD7"}},{"Branch":{"value":"FIELD~AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA-","children":[{"ForgettenSubtree":{"value":"FIELD~WdWWbJavfsrVydKRjWWC0QKyxn9rdl6iisJDcatPk77C"}},{"ForgettenSubtree":{"value":"FIELD~a28U5q8mJxhr-KVeR03PbvP2zoGvVg2ocexR62cBcanN"}},"Empty"]}},{"Branch":{"value":"FIELD~AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA-","children":[{"ForgettenSubtree":{"value":"FIELD~lf76E5t7luYWTs4hhs3Fhwd6XNhypSL_1MKH4eJU6u_T"}},{"ForgettenSubtree":{"value":"FIELD~9vP_o7cAgIbihRvlRrOrp2E6L5S5TNDYjLmf8DcnNBX5"}},"Empty"]}}]}},"common":{"poly_commits":"FIELD~AAAAAAAAAAD7","all_evals_digest":"FIELD~HE7xxRlER6dA-1s93kzScegDxQTi-H-PpEL2vIBhgaXb","payload_byte_len":0,"num_storage_nodes":5,"multiplicity":1},"recipient_key":"BLS_VER_KEY~bQszS-QKYvUij2g20VqS8asttGSb95NrTu2PUj0uMh1CBUxNy1FqyPDjZqB29M7ZbjWqj79QkEOWkpga84AmDYUeTuWmy-0P1AdKHD3ehc-dKvei78BDj5USwXPJiDUlCxvYs_9rWYhagaq-5_LXENr78xel17spftNd5MA1Mw5U"}},"state_cert":null}],"qc":{"data":{"leaf_commit":"COMMIT~zDQj9T-fNklWU9VpkdYCE3n6vsf697rt8_uHDwgyving","epoch":null,"block_number":null},"vote_commitment":"COMMIT~Wwdd_O5aIONcAtJDlPmi9Gnv4MCrqU_Qaik9A4yz-gU_","view_number":13,"signatures":["BLS_SIG~2YwGf9PcBz0fkcg92TXTQYju867TkrZXqbSuU3HQfShjpqJo54AyY2QOfH-xYpgJaH8s6-lyxjf8O4efHk-yrgE",{"order":"bitvec::order::Lsb0","head":{"width":64,"index":0},"bits":5,"data":[27]}],"_pd":null},"block_size":0}}}`
```

Comment:
Looks like these samples are used for testing? Move them to the test file?

Reply:
Thanks, will do

Reply:
It was giving me typo issues in the CI, I think its fine to be in consts for now